test/unit/status/myDomainResolverTest.ts

```diff
@@ -78,4 +78,69 @@ describe('myDomainResolver', () => {
       expect(e.name).to.equal('MyDomainResolverTimeoutError');
     }
   });
+
+  it('should resolve localhost', async () => {
+    const options: MyDomainResolver.Options = {
+      url: new URL('http://ghostbusters.internal.salesforce.com'),
```

Comment:
is this url format still in use?  Other tests have examples with both `localhost` and a port
https://github.com/forcedotcom/sfdx-core/blob/5f17bf5635475963cbca85f6a4560d7f6aa39141/test/unit/authInfoTest.ts#L70